app/code/Mykhailok/SupportChat/view/frontend/web/js/view/chat.js

```diff
@@ -0,0 +1,72 @@
+define([
+    'jquery',
+    'ko',
+    'uiComponent',
+    'mykhailokSupportChatModelMessage',
+    'mykhailokSupportChatActionSave'
+], function ($, ko, Component, messageModel, saveMessage) {
+    'use strict';
+
+    return Component.extend({
+        messageValue: ko.observable(),
+        messages: messageModel().data,
```

Comment:
@mykhailok-dv Use properly formatted data from the customerData (it is observable by default).

Reply:
You said that this model can be left or I should change the format of the getSectionData() method.